.gitignore

```diff
@@ -84,3 +84,9 @@ packaging/win32_3rdparty/bin
 packaging/win32_3rdparty/share
 packaging/win32_3rdparty/lib
 packaging/win32_3rdparty/include
+test/src/unittest/*.npy
```

Comment:
This should be updated because all tests were moved to the `unittests`folder.

Reply:
In any case, we should rather not generate junk files inside `test/src/unit` test, but use a proper temp folder instead. I've created an issue for that (#861), so we can remove tose ignore lines here.